Jenkinsfile

```diff
@@ -99,6 +99,8 @@ def runTests(dockerVersion, pythonVersion, baseImage) {
                       --privileged \\
```

Comment:
Do we actually need `--privileged` for this container? I see it bind-mounts `docker.sock`, which means that all `docker` commands will actually run against the docker daemon running on the host (not a `dockerd` daemon running inside the container (which _would_ require privileged)

Reply:
Actually wondering if we need a container here at all, because it's running a container, just to run a script, that uses the `docker` CLI inside the container to start new containers on the host 🤔